SDK/ViewControllers/Settings/SettingsViewController.swift

```diff
@@ -211,9 +211,21 @@ extension SettingsViewController: UITableViewDataSource {
             return cell
         case .purchases:
             let cell = tableView.dequeueReusableCell(withIdentifier: "CellSubtitle", for: indexPath)
-            cell.textLabel!.text = item.title
-            cell.detailTextLabel!.text = ""
-            cell.detailTextLabel!.textColor = .lightGray
+            if UserDefaults.standard.subscriptionExpirationDate == nil {
+                UserDefaults.standard.subscriptionExpirationDate = Date()
```

Comment:
Is it documented behavior that a `nil` expiration date means "now"? How and when is that value updated and why would it be `nil` if there is a valid subscription?

Reply:
The value is updated when a subscription is made, we could find a nil value if the user bought a subscription, deleted the app and installed again, this is a safe check.

Reply:
Instead of showing the current date, just hide the row or show expiration time unavailable

Reply:
Done, we don't show the subtitle with the time left